custom_components/tuya_local/devices/moes_wht009_thermostat.yaml

```diff
@@ -0,0 +1,226 @@
+name: Moes thermostat WHT-009
```

Comment:
This top level name should be without branding (branding belongs under products, as often configs may match multiple devices).